sources/meta-yoe/conf/distro/yoe.inc

```diff
@@ -91,7 +91,7 @@ require conf/distro/include/no-static-libs.inc
 require conf/distro/include/yocto-uninative.inc
 require conf/distro/include/security_flags.inc
 
-#INHERIT += "${@bb.utils.contains('BUILD_ARCH', 'ppc64le', '', 'uninative', d)}"
```

Comment:
curious -- why is this only done for ppc64le?

Reply:
yocto dpes not publish uninative tarballs for ppc64le hosts thats why its disabled